tests/ReflectorFactory.php

```diff
@@ -19,11 +19,13 @@ public static function create(string $cacheDir): QueryReflector
     {
         // handle defaults
         if (false !== getenv('GITHUB_ACTION')) {
```

Comment:
I think the cache-file below `$cacheDir.'/.phpunit-phpstan-dba-'.$reflector.'.cache';` needs to be changed, so it contains the database type (we need separate cache files for pgsql)